adapter/internal/messaging/azure_listener.go

```diff
@@ -29,7 +29,7 @@ import (
 
 const (
 	componentName                = "adapter"
-	subscriptionIdleTimeDuration = time.Duration(72 * time.Hour)
+	subscriptionIdleTimeDuration = "P0Y0M3DT0H0M0S"
```

Comment:
What does this value stand for? Shall we add a comment?

Reply:
This is a time in ISO 8601 format actually, does n't variable name indicate the meaning behind it?